src/mongod_helpers.py

```diff
@@ -108,6 +108,16 @@ def _is_primary(self):
         client = self.client(standalone=True)
         return client.is_primary
 
+    def primary_step_down(self) -> None:
+        """Steps primary replica down, enabling one of the secondaries to become primary."""
+        client = self.client()
+        try:
+            config = {}
+            config["stepDownSecs"] = "60"
```

Comment:
Could this be combined to the following: 
```python
config = {"stepDownSecs": "60"}
```

Reply:
🤦 yes this is quite right, thanks for the catch